common/pathsofinterest.go

```diff
@@ -25,6 +25,10 @@ import (
 const (
 	// DefaultPlanFile defines default name for plan file
 	DefaultPlanFile = types.AppNameShort + ".plan"
+	// DefaultSourceFolder defines default name for plan file
+	DefaultSourceFolder = types.AppNameShort + "-input"
```

Comment:
Lets not have a default for source, it might not be obvious

Reply:
done!